src/Nethermind/Nethermind.Merge.Plugin/Data/Result.cs

```diff
@@ -21,12 +21,12 @@ namespace Nethermind.Merge.Plugin.Data
 {
     public class Result
     {
-        public static readonly Result Success = new Result() {Value = true};
-        public static readonly Result Fail = new Result() {Value = false};
+        public static readonly Result OK = new() {success = true};
+        public static readonly Result Fail = new() {success = false};
 
-        public static implicit operator Result(bool value) => value ? Success : Fail;
+        public static implicit operator Result(bool value) => value ? OK : Fail;
 
         [JsonPropertyName("success")]
```

Comment:
There was a [JsonPropertyName("success")] here it did serialize fine for me, so I am not sure what was the matter? Also maybe uppercase?